selfdrive/car/chrysler/chryslercan.py

```diff
@@ -48,10 +48,10 @@ def create_lkas_command(packer, apply_steer, moving_fast, frame):
   return packer.make_can_msg("LKAS_COMMAND", 0, values)
 
 
-def create_wheel_buttons(packer, frame, cancel=False):
+def create_wheel_buttons(packer, counter, cancel=False):
   # WHEEL_BUTTONS (571) Message sent to cancel ACC.
   values = {
     "ACC_CANCEL": cancel,
-    "COUNTER": frame % 10
+    "COUNTER": counter
```

Comment:
Since we are adding 1 to the `WHEEL_BUTTONS`'s `COUNTER`, we need a mod 16.  This is in case the `COUNTER` is 15 and when we add 1, it should be 0 and not 16 since the `COUNTER` is 0-15

```suggestion
    "COUNTER": counter % 16
```

Reply:
since counter itself is a 4 bit signal, it is not necessary to do the % 16, the signal will wrap around when packed on to the CAN bus. 16 will be went as 0 and 17 as 1 and so on.. but either way it is correct.